SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -156,6 +156,7 @@ void USpatialReceiver::OnAddComponent(const Worker_AddComponentOp& Op)
 	case SpatialConstants::ALWAYS_RELEVANT_COMPONENT_ID:
 	case SpatialConstants::CLIENT_RPC_ENDPOINT_COMPONENT_ID_LEGACY:
 	case SpatialConstants::SERVER_RPC_ENDPOINT_COMPONENT_ID_LEGACY:
+	case SpatialConstants::SERVER_TO_SERVER_ENDPOINT_COMPONENT_ID_LEGACY:
```

Comment:
Don't you need to make this change to the ring-buffer RPC setup also?

Reply:
It doesn't _need_ to happen, at least not necessarily as part of this PR. I am going to make a follow up PR to do the same for ring buffer RPCs, and speaking to @samiwh offline it sounds like the best plan for that is to reuse the new server to server component as both approaches use commands. To prepare for that, I will rename the component from something that says legacy to something that says commands

Reply:
Ring buffers have another copy of the command they use when turned on. We were discussing yesterday that there shouldn't be two of these and we should just decouple the commands from other rpcs, but that it can be done as a separate PR to make it easier to review. However I think we shouldn't call the command based RPCs legacy.

Reply:
https://improbableio.atlassian.net/browse/UNR-2904 for the follow up work